src/main/java/org/example/ConnectionSingleton.java

```diff
@@ -0,0 +1,20 @@
+package org.example;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionSingleton {
+    private static Connection connection;
+    private static final String NAME = "postgres";
```

Comment:
It's better to store the link and credentials separately in a separate file and don't publish in github